components/browser/storage-sync/README.md

```diff
@@ -2,6 +2,11 @@
 
 A syncable implementation of `concept-storage` backed by [application-services' Places lib](https://github.com/mozilla/application-services).
 
+## Before using this component
+Products sending telemetry and using this component *must request* a data-review following [this process](https://wiki.mozilla.org/Firefox/Data_Collection).
```

Comment:
I wonder if it's worth making this configurable—say, with a Boolean that we check in `RustPlacesConnection::send{History, Bookmarks}Ping`—so that applications that don't need telemetry won't send it. (We'll need to revisit this when we start sending telemetry for storage, not just Sync). I think requiring data review to use this component with Glean is fine now, though!

Reply:
In general, if this component is used in an application that doesn't have Glean enabled, nothing will get collected (so no need for special casing or other options). However, if the product uses Glean then this component will automatically contribute with its own metrics. We might consider adding more flexibility here if we stumble upon the case of a component supporting Glean, being embedded in a product using Glean, with the product sending telemetry but not wanting the component to send it.